tstest/natlab/natlab.go

```diff
@@ -191,17 +191,22 @@ type routeEntry struct {
 	iface  *Interface
 }
 
-// NewMachine returns a new Machine without any network connection.
-// The name is just for debugging and need not be globally unique.
-// Use Attach to add networks.
-func NewMachine(name string) *Machine {
-	return &Machine{name: name}
-}
+// A PacketHandler is a function that can process packets.
+type PacketHandler func(p []byte, dst, src netaddr.IPPort) (drop bool)
 
 // A Machine is a representation of an operating system's network stack.
 // It has a network routing table and can have multiple attached networks.
```

Comment:
// The zero value is valid, but lacks any networking capabilities until Attach is called.